src/material/datepicker/datepicker.md

```diff
@@ -252,7 +252,20 @@ By default the `MomentDateAdapter` will creates dates in your time zone specific
 @NgModule({
   imports: [MatDatepickerModule, MatMomentDateModule],
   providers: [
-    { provide: MAT_MOMENT_DATE_ADAPTER_OPTIONS, useValue: { useUtc: true } }
+    {provide: MAT_MOMENT_DATE_ADAPTER_OPTIONS, useValue: {useUtc: true}}
+  ]
+})
+```
+
+By default the `MomentDateAdapter` will parse dates in a forgiving way. This may result in dates
```

Comment:
can you link to the section in the moment docs that talks about strict parsing/ so people can easily hop over for more info

Reply:
Done